<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Label_tool2Class</class>
 <widget class="QMainWindow" name="Label_tool2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1757</width>
    <height>1260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Label_tool2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>281</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QRadioButton" name="radio_point">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>102</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>point</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_rect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>102</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Rect</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_open">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Open File</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>120</width>
       <height>231</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>118</width>
        <height>229</height>
       </rect>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>104</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLineEdit" name="label_edit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>330</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="b_savedir">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>430</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save_dir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_saveimg">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>500</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save_img</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_saved">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>510</y>
       <width>79</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>saved</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>761</width>
      <height>681</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="mylabel" name="img_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>531</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>520</y>
       <width>621</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_previous">
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>124</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_next">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>125</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>480</y>
       <width>691</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="text">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>830</y>
      <width>931</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>Label</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1757</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>mylabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Label_tool2.qrc"/>
 </resources>
 <connections/>
</ui>
